src/moonlink_connectors/src/rest_ingest/json_converter.rs

```diff
@@ -107,6 +110,20 @@ impl JsonToMoonlinkRowConverter {
                     Err(JsonToMoonlinkRowError::TypeMismatch(field.name().clone()))
                 }
             }
+            DataType::Decimal128(precision, scale) => {
+                if let Some(s) = value.as_str() {
+                    convert_decimal_to_row_value(s, *precision, *scale)
+                        .map_err(|_| JsonToMoonlinkRowError::InvalidValue(field.name().clone()))
```

Comment:
This is sth I feel really weird... We have nice error enums for decimal conversion, but certain errors are missed during error propagation.